src/core/storage/utils.ts

```diff
@@ -1,14 +1,14 @@
 import { Capabilities, Delegation } from '@ucanto/interface';
 import { lookup } from 'mime-types';
-import * as Proof from '@web3-storage/w3up-client/proof';
+import * as Proof from '@storacha/client/proof';
 
 /**
  * Parses a delegation from a base64 encoded CAR file
  * @param data - The base64 encoded CAR file
  * @returns The parsed delegation
  */
 export async function parseDelegation(data: string): Promise<Delegation<Capabilities>> {
-  const proof = await Proof.parse(data);
+  const proof = await Proof.parse(data.replaceAll('\n', ''));
```

Comment:
```suggestion
  const proof = await Proof.parse(data.trim());
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim

We should move that into the `Proof.parse()` function...